pkgs/development/python-modules/aioinflux/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, pytz
+, aiohttp
+, ciso8601
+, pandas
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "aioinflux";
+  version = "0.9.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "gusutabopb";
+    repo = "aioinflux";
+    rev = "v${version}";
+    sha256 = "0cvzkd05i8bzh76m75s7na2gb0kh5msyyz60ajxpj2by9x6qkxmc";
+  };
+
+  propagatedBuildInputs = [ aiohttp pytz ciso8601 pandas ];
+
+  # Tests require local networking so won't run in the sandbox
+  doCheck = false;
```

Comment:
please try to do some tests, you can always scope tests using `pytest -k 'not some_test_string'`

Reply:
Yes, I agree. I've looked upstream but haven't found any suitable test filters yet. I might contribute a few tests so we can verify the package requirements are right.

I understand @FRidh isn't in a hurry to merge any python packages until we solve the scalability problem. So this isn't a super high priority for me, but I am still interested in getting it merged at some point. 

Just letting you know I haven't jumped ship!

Reply:
it's certainly a salability issue for one person, but not as much of an issue for many

Reply:
It's more solved than I thought then. 
I wonder if we could use mypy for a minimal check for all python packages. It can find if there are missing dependencies.

Reply:
looks like mypy package logic differs from standard python mechanisms and makes assumptions as to where the packages should be located. Probably not compatible with nix

https://mypy.readthedocs.io/en/latest/running_mypy.html#how-imports-are-found

Reply:
Yeah, I've had trouble with the logic, and spent alot of time in that section of the mypy docs. 

I guess nothing beats actually running the code in the interpreter, and any python package worth packaging will have good test coverage anyway.

Reply:
you can also do `pythonImportsCheck = [ "foo.bar" ];` to essentially run `python -c 'import foo.bar'` 

I use it extensively https://github.com/NixOS/nixpkgs/blob/3140fa89c51233397f496f49014f6b23216667c2/pkgs/tools/admin/azure-cli/default.nix#L168 to ensure that the azure namespace isn't broken by a rogue __init__.py file